test/data/ReqMgr/requests/DMWM/StepChain_MC.json

```diff
@@ -1,74 +1,82 @@
-    "SubscriptionPriority": "Low",
-    "CustodialSubType" : "Move",
-    "BlockCloseMaxWaitTime" : 72000,
-    "BlockCloseMaxFiles" : 500,
-    "BlockCloseMaxEvents" : 200000000,
-    "BlockCloseMaxSize" : 5000000000000
+{
+    "assignRequest": {
+        "Dashboard": "Dashboard-OVERRIDE-ME", 
+        "GracePeriod": 300, 
+        "TrustPUSitelists": true,
+        "MaxRSS": 2300000, 
+        "MaxVSize": 20000000, 
+        "MergedLFNBase": "/store/backfill/1", 
+        "ProcessingVersion": 19, 
+        "SiteWhitelist": [
+            "SiteWhitelist-OVERRIDE-ME"
+        ], 
+        "SoftTimeout": 129600, 
+        "Team": "Team-OVERRIDE-ME", 
+        "UnmergedLFNBase": "/store/unmerged"
+    }, 
+    "createRequest": {
+        "AcquisitionEra": "Integ_Test", 
+        "CMSSWVersion": "CMSSW_8_0_21", 
+        "Campaign": "Campaign-OVERRIDE-ME", 
+        "Comments": ["No input; PU in Step2; Step1 using diff CMSSW/Scram; Keep only AODSIM; Automatic splitting 200 EpJ, 2 LpJ",
+                     "Processed dataset defined during creation; TrustPU flag enabled"],
+        "ConfigCacheUrl": "https://cmsweb.cern.ch/couchdb", 
+        "CouchDBName": "reqmgr_config_cache", 
+        "DbsUrl": "https://cmsweb-testbed.cern.ch/dbs/int/global/DBSReader/", 
+        "GlobalTag": "80X_mcRun2_asymptotic_2016_TrancheIV_v6", 
+        "Group": "DATAOPS", 
+        "Memory": 2300,
+        "MergedLFNBase": "/store/backfill/1", 
+        "PrepID": "TEST-HIG-RunIISummer15wmLHEGS-00744", 
+        "PrimaryDataset": "DYJetsToLL_Pt-50To100_TuneCUETP8M1_13TeV-amcatnloFXFX-pythia8", 
+        "ProcessingString": "StepChain_GenSimDigiReco_TEST", 
+        "ProcessingVersion": 1, 
+        "RequestPriority": 216000, 
+        "RequestString": "RequestString-OVERRIDE-ME", 
+        "RequestType": "StepChain", 
+        "Requestor": "amaltaro", 
+        "ScramArch": "slc6_amd64_gcc530", 
+        "SizePerEvent": 250, 
+        "Step1": {
+            "CMSSWVersion": "CMSSW_7_1_25_patch2", 
+            "ConfigCacheID": "526ca0745cd309b7cfef9f23b3d43acb", 
+            "EventsPerLumi": 100, 
+            "GlobalTag": "MCRUN2_71_V1::All", 
+            "KeepOutput": false, 
+            "PrepID": "TEST-Step1-RunIISummer15wmLHEGS-00744", 
+            "RequestNumEvents": 20000, 
+            "ScramArch": "slc6_amd64_gcc481", 
+            "Seeding": "AutomaticSeeding", 
+            "SplittingAlgo": "EventBased", 
+            "StepName": "GENSIM"
+        }, 
+        "Step2": {
+            "CMSSWVersion": "CMSSW_8_0_21", 
+            "ConfigCacheID": "f7e311f2c6b5a0884faea133990edcbf", 
+            "GlobalTag": "80X_mcRun2_asymptotic_2016_TrancheIV_v6", 
+            "InputFromOutputModule": "RAWSIMoutput", 
+            "InputStep": "GENSIM", 
+            "KeepOutput": false, 
+            "MCPileup": "/Neutrino_E-10_gun/RunIISpring15PrePremix-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v2-v2/GEN-SIM-DIGI-RAW", 
```

Comment:
I just noticed this awesome Stepchain template is using the evil PU sample :( I'm updating it soon.